src/main/java/net/sacredlabyrinth/phaed/simpleclans/utils/ChatUtils.java

```diff
@@ -162,4 +167,14 @@ public static void applyLastColorToFollowingLines(@NotNull List<String> lines) {
             }
         }
     }
+
+    public static String formatCurrency(double value) {
+        return plugin.getSettingsManager().format(ECONOMY_DECIMAL_FORMAT_PATTERN, value);
+    }
```

Comment:
Why not use `plugin.getSettingsManager().format(ECONOMY_DECIMAL_FORMAT_PATTERN, value);` directly? 👀

Reply:
Does format() belong in SettingsManager?